scene/gui/range.cpp

```diff
@@ -326,6 +326,9 @@ void Range::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "allow_greater"), "set_allow_greater", "is_greater_allowed");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "allow_lesser"), "set_allow_lesser", "is_lesser_allowed");
 
+	// Default values are non-deterministic, override for doc generation purposes.
```

Comment:
This is not true, that's for RNG.
```suggestion
	// Default value would be 0.01 otherwise, override for doc generation purposes.
```

Reply:
Done